File: 백준/Silver/17219. 비밀번호 찾기/비밀번호 찾기.py
import sys

n, t = map(int, sys.stdin.readline().split())
store = {}

for _ in range(n):
    [site, pw] = list(sys.stdin.readline().split())
    store[site] = pw

for _ in range(t):
    site = sys.stdin.readline().strip()
    print(store.get(site))
    

File: 백준/Silver/17219. 비밀번호 찾기/비밀번호 찾기.js
const input = require("fs")
  .readFileSync("/dev/stdin")
  .toString()
  .trim()
  .split("\n");

const cases = input[0];
const [n, t] = cases.split(" ").map(Number);
let dict = {};

for (let i = 1; i <= n; i++) {
  let [site, pw] = input[i].split(" ");
  dict[site] = pw;
}

for (let i = n + 1; i <= n + t; i++) {
  let search = input[i];
  console.log(dict[search]);
}
